lib/dradis/plugins/content_service/content_blocks.rb

```diff
@@ -0,0 +1,9 @@
+module Dradis::Plugins::ContentService
+  module ContentBlocks
+    extend ActiveSupport::Concern
+
+    def all_content_blocks
+      @project.content_blocks
```

Comment:
I don't think we should query the attribute here. Let's follow the same structure that we already have with Boards and use an AR finder instead please.